field_parser.go

```diff
@@ -207,7 +207,9 @@ func (ps *tagBaseFieldParser) ComplementSchema(schema *spec.Schema) error {
 	extensionsTag := ps.tag.Get(extensionsTag)
 	if extensionsTag != "" {
 		structField.extensions = map[string]interface{}{}
+		extensionsTag := strings.ReplaceAll(extensionsTag, `,,`, `{{escape-comma}}`)
 		for _, val := range strings.Split(extensionsTag, ",") {
```

Comment:
Adding double-quoted is an ugly hack. 

The solution is to line instead of split by comma by respection open/close brackets.

Reply:
I agree. double-comma was a quick fix. I'll try the solution you suggested

Reply:
Changed as suggested